inlong-sort/sort-sql/README.md

```diff
@@ -0,0 +1,50 @@
+
+# Description
+
+## overview
+Inlong-sort is used to extract data from different source systems, then transforms the data and finally loads the data into different storage systems.
```

Comment:
the overview part should add more description for `sort-sql`.

Reply:
make sense, will update it.